src/bridge/MatrixHandler.ts

```diff
@@ -30,6 +31,12 @@ const MSG_PMS_DISABLED_FEDERATION = "[Bridge] Sorry, PMs are disabled on this br
 const DEFAULT_EVENT_CACHE_SIZE = 4096;
 /* Length of the source text in a formatted reply message */
 const REPLY_SOURCE_MAX_LENGTH = 32;
+// How many seconds needs to pass between a message and a reply to it to switch to the long reply format
+const DEFAULT_SHORT_REPLY_TRESHOLD_SECONDS = 5 * 60;
+// Format of replies sent shortly after the original message
+const DEFAULT_SHORT_REPLY_TEMPLATE = "$NICK: $REPLY";
+// Format of replies sent a while after the original message
+const DEFAULT_LONG_REPLY_TEMPLATE = "<$NICK> \"$ORIGINAL\" <- $REPLY";
```

Comment:
I wish we had a default config somewhere rather than having consts at the top of the file, but I see I have also done this.

Reply:
Yeah, I'm not super fond of it, but everything seems to be kept in sync manually between `config.sample.yaml` and the code – sometimes in `consts` like this, sometimes in good old magic numbers. I wouldn't mind fixing that also, but it's probably a better fit for another day :)

Reply:
In the teams / github bridges we have magic code to generate a config sample from defaults, with comments. I should port that to the common bridge lib some day.